### PR TITLE
schedule: fix priority bucket number

### LINKED ISSUE #5595: `RandBuckets` panic due to the new priority level
### What did you do?

Daily testing.

### What did you expect to see?

All tests passed

### What did you see instead?
```
[2022/10/13 13:57:47.871 +08:00] [FATAL] [log.go:72] [panic] [recover={}] [stack="[github.com/tikv/pd/pkg/logutil.LogPanic\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/pd/pkg/logutil/log.go:72\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:884\nruntime.goPanicIndex\n\t/usr/local/go/src/runtime/panic.go:113\ngithub.com/tikv/pd/server/schedule.(*RandBuckets).PutOperator\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/pd/server/schedule/waiting_operator.go:60\ngithub.com/tikv/pd/server/schedule.(*OperatorController).AddWaitingOperator\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/pd/server/schedule/operator_controller.go:293\ngithub.com/tikv/pd/server/cluster.(*coordinator).runScheduler\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/pd/server/cluster/coordinator.go:80](http://github.com/tikv/pd/pkg/logutil.LogPanic/n/t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/pd/pkg/logutil/log.go:72/nruntime.gopanic/n/t/usr/local/go/src/runtime/panic.go:884/nruntime.goPanicIndex/n/t/usr/local/go/src/runtime/panic.go:113/ngithub.com/tikv/pd/server/schedule.(*RandBuckets).PutOperator/n/t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/pd/server/schedule/waiting_operator.go:60/ngithub.com/tikv/pd/server/schedule.(*OperatorController).AddWaitingOperator/n/t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/pd/server/schedule/operator_controller.go:293/ngithub.com/tikv/pd/server/cluster.(*coordinator).runScheduler/n/t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/pd/server/cluster/coordinator.go:804)
```

### What version of PD are you using (`pd-server -V`)?
master

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

Issue Number: Close #5595

### What is changed and how does it work?

We add a new priority level without changing the bucket number, which causes panic.

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None
```

### CODE REVIEW

File: server/schedule/waiting_operator_test.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ func TestRandBuckets(t *testing.T) {
 	re := require.New(t)
 	rb := NewRandBuckets()
 	addOperators(rb)
-	for i := 0; i < 3; i++ {
+	for i := 0; i < 4; i++ {
```
Comment (bufferflies) [inline server/schedule/waiting_operator_test.go:30]: how about using `len(PriorityWeight)`

File: server/schedule/waiting_operator_test.go [hunk 2]
```diff
@@ -49,13 +50,18 @@ func addOperators(wop WaitingOperator) {
...
+	op.SetPriorityLevel(core.Urgent)
+	wop.PutOperator(op)
 }
 
 func TestListOperator(t *testing.T) {
 	re := require.New(t)
 	rb := NewRandBuckets()
 	addOperators(rb)
-	re.Len(rb.ListOperator(), 3)
+	re.Len(rb.ListOperator(), 4)
```
Comment (bufferflies) [inline server/schedule/waiting_operator_test.go:64]: ditto

**Review by bufferflies — APPROVED** [review]
LGTM

**Review by CabinfeverB — APPROVED** [review]

**Pull request merged**